Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -3150,7 +3150,7 @@ void finishMolProcessing(RWMol *res, bool chiralityPossible, bool sanitize,
   // perceive chirality, then remove the Hs and sanitize.
   //
   const Conformer &conf = res->getConformer();
-  if (chiralityPossible) {
+  if (chiralityPossible || conf.is3D()) {
```

Comment:
Is it documented that chirality is determined from the first conformer (if it's 3D?) it very well could be that a multi conformer molecule has more than one stereo chemistry, even though they really shouldn't.